app/views/taxon_ranges/_form_fields.html.haml

```diff
@@ -0,0 +1,27 @@
+.row.stacked
+  .col-xs-6
+    = f.text_field :taxon_id
+    - if f.object.range.file?
+      .field
+        = link_to f.object.range.url, f.object.range.url
+        = link_to_function t(:replace_parentesis), "$(this).parents('.field').after('#{f.file_field(:range)}'); $(this).parents('.field').hide()", :class => "small"
+        = f.check_box :range_delete, label: t(:delete_range), label_after: true
+    - else
+      = f.file_field :range
+    = f.form_field :iucn_relationship, :label => false do
+      = label_tag :is_this_an_iucn_redlist_range_map_and_if_not_why, t(:is_this_an_iucn_redlist_range_map_and_if_not_why)
+      = f.radio_button :iucn_relationship, TaxonRange::IUCN_REDLIST_MAP, label: t(:this_is_an_iucn_redlist_range_map)
+      = f.radio_button :iucn_relationship, TaxonRange::IUCN_REDLIST_MAP_HAS_ISSUES, label: t(:iucn_redlist_range_map_for_this_taxon_has_issues)
+      = f.radio_button :iucn_relationship, TaxonRange::NOT_ON_IUCN_REDLIST, label: t(:taxon_not_in_the_iucn_redlist)
```

Comment:
Since `iucn_relationship` can be null, I think we need a fourth option here for that, like "other" or "unknown" or something. In addition to "unknown" can't a range be for a taxon that *is* on the Red List but the range didn't come from the IUCN?

Reply:
My vision was that we should work towards every taxon range having one of these 3 values:
IUCN_REDLIST_MAP
IUCN_REDLIST_MAP_HAS_ISSUES
NOT_ON_IUCN_REDLIST
So while we display unknown for the many existing records where this is unknown, we want to encourage curators to choose one of those 3 if they are going to update that attribute.
For the range is on the Red List but the range didn't come from the IUCN case it would be IUCN_REDLIST_MAP_HAS_ISSUES because there's still some reason why someone felt the IUCN range was unsuitable to display and therefore opted for an alternative.

Reply:
Ah, that wasn't clear to me. Could we change that text to read, "This is not an IUCN Red List range map because that map has problems" or something? Maybe the last could be "This is not an IUCN Red List range map because this taxon is not on the IUCN Red List"?

Reply:
I changed it to that but I'm going with "unsuitable" rather than "has problems" to make it more subjective to iNat's purposes